reana_commons/config.py

```diff
@@ -407,3 +407,11 @@ def kubernetes_node_label_to_dict(node_label):
 Example: ``REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES="finished,failed"``
 would keep jobs that terminated successfully and jobs that failed.
 """
+
+REANA_COMPUTE_BACKENDS = {
+    "kubernetes": "Kubernetes",
+    "htcondor": "HTCondor",
+    "slurm": "Slurm",
+}
+
```

Comment:
Extremely minor: I guess empty line is not needed here

Reply:
Fixed!